addon/templates/components/bs4/bs-form/element/layout/horizontal.hbs

```diff
@@ -0,0 +1,12 @@
+{{#if hasLabel}}
+  {{component labelComponent labelClass=horizontalLabelGridClass}}
+  <div class="{{horizontalInputGridClass}}">
+    {{yield}}
+    {{component errorsComponent}}
+  </div>
+{{else}}
+  <div class="{{horizontalInputGridClass}} {{horizontalInputOffsetGridClass}}">
+    {{yield}}
+    {{component errorsComponent}}
+  </div>
+{{/if}}
```

Comment:
Ok, good to know. Thought it worked like this at some point, but might confuse it...